Add a main README.md introducing each service (#4)
beemobot/cafe: README.md

### Cafe

> The Beemo Cafe is a mono-repo containing all the public microservices that Beemo will, or is utilizing. In this mono-repo contains the following services:

Reviewer: I'd just say that the stuff here actually powers Beemo, not that it may or may not be currently powering it now or in the future, that sounded a bit confusing. Also, a bit of grammar at the end 😄

```suggestion
The Beemo Cafe is a monorepo containing all the public microservices that are powering Beemo. The repo consists of the following services:

```

|                              Name                             |     Purpose     |                                     Description                                     |
|:-------------------------------------------------------------:|:---------------:|:-----------------------------------------------------------------------------------:|
|   [Latte](https://github.com/beemobot/cafe/tree/main/latte)   |     Commons     |  A set of Java-Kotlin commons that are used all over Beemo's Java-Kotlin services.  |

> |    [Milk](https://github.com/beemobot/cafe/tree/main/milk)    |   Public Logs   |            Handles storing and public viewing of logs such as raid logs.            |

Reviewer: Since those are the only logs it stores under the new scope :D

```suggestion
|    [Milk](https://github.com/beemobot/cafe/tree/main/milk)    |   Public Logs   |            Handles storing and public viewing of raid logs.                         |
```
